### PR TITLE
Autowire DiagramOrchestrator and DataLayer

### PR DESCRIPTION
**Pull request by eliskovets**
Hi guys, 
I've made some changes to decouple datalayer from diagramOrchestrator. 
Please take a look and merge it if you like it. 
@ceckoslab it also removes the registry parameter from constructors and uses entityManager instead to get the DB connection.

### CODE REVIEW

**Comment by ceckoslab** [comment]
@eliskovets  YAY! 

It looks good. I will check it a bit more in details but I am happy to see different parts of the project getting decoupled :)

File: config/services.yaml [hunk 1]
```diff
@@ -30,3 +30,11 @@ services:
...
     # add more service definitions when explicit configuration is needed
     # please note that last definitions always *replace* previous ones
+    App\BasicRum\Layers\DataLayer:
+        public: true
+    App\BasicRum\Layers\DataLayer\Query\Runner:
+        calls:
+            - [setConnection, ['@doctrine.orm.entity_manager']]
+        public: true
+    App\BasicRum\DiagramOrchestrator:
+        public: true
```
Comment (eliskovets) [inline config/services.yaml:40]: I have to expose these services to public as we need to use them in tests, but i believe it should be a better way to make it. If you know it, please tell me and I will adjust the code accordingly.

**Comment by ceckoslab** [comment]
Hello @eliskovets 

I am about to merge the PR. It looks really good and it's much better that what we had previously. 

I noticed something small. I run the code fixer from the app container:
```
vendor/bin/php-cs-fixer fix --config=./.php_cs
```

It did very small corrections over the code but it wiped some of your opening function comments. Do you think that you may take a look and add rules in our php_cs file so your comments don't get wiped?

**Comment by eliskovets** [comment]
Hi @ceckoslab 
Thank you for noticing this. I will prepare required changes in the `.php_cs` file.

**Comment by eliskovets** [comment]
Hi @ceckoslab ,
It looks like it wouldn't be possible to reconfigure `phpdoc_add_missing_param_annotation` to keep arguments in the list of params in phpdocs in case if their type/class specified in definition of function, as we've got `no_superfluous_phpdoc_tags` config from `@Symfony` which conflicts with `phpdoc_add_missing_param_annotation => ['only_untyped' => false]`. I haven't specified any descriptions for parameters in phpdoc, so they've been removed by php-cs-fixer which is probably a right behaviour.

I've also reconfigured my IDE to reformat code based on the .php_cs configuration, so it should not happen in future. I will commit changes made by php-cs-fixer in a few minutes.

**Pull request merged**